std/prettier/main_test.ts

```diff
@@ -226,20 +225,35 @@ test(async function testPrettierPrintToStdout(): Promise<void> {
 
   const { stdout } = await run([...cmd, file0]);
   // The source file will not change without `--write` flags.
-  assertEquals(await getSourceCode(file0), "console.log (0)" + EOL);
+  assertEquals(
+    await getSourceCode(file0),
+    `console.log (0)
+`
```

Comment:
this backtick string pattern is used here https://github.com/nstott/deno/blob/reenable-stdlib-tests/std/prettier/main_test.ts#L190-L191 and seems to work on all arches
testing for EOL on windows builds is odd, for example in this test

`assertEquals(await getSourceCode(file0), "console.log (0)" + EOL);`

`"console.log (0)" + EOL` has a different value on windows builds than on linux/mac, but the contents of the file don't change.